### PR TITLE
Fixed letters being cut off in Admin UI cards

### PR DESCRIPTION
**Pull request by Vultraz**
Before (notice the bottom of the G is cut off):
image

After:
image

Space between the two lines is text is also increased as a result (it's too close with a `line-height` of 1.1), which I think looks better.

### CODE REVIEW

File: packages/app-admin-ui/client/pages/Home/components.js [hunk 1]
```diff
@@ -76,7 +75,7 @@ export const Name = styled.span(
   font-size: 1.1em;
```
Comment (MadeByMike) [inline packages/app-admin-ui/client/pages/Home/components.js]:
```suggestion
  font-size: 1.1rem;
```

File: packages/app-admin-ui/client/pages/Home/components.js [hunk 2]
```diff
@@ -76,7 +75,7 @@ export const Name = styled.span(
...
   font-weight: 500;
   max-width: 100%;
-  overflow: hidden;
+  overflow-x: hidden;
```
Comment (MadeByMike) [inline packages/app-admin-ui/client/pages/Home/components.js]:
```suggestion
  overflow: hidden;
```

File: packages/app-admin-ui/client/pages/Home/components.js [hunk 3]
```diff
@@ -76,7 +75,7 @@ export const Name = styled.span(
...
   text-overflow: ellipsis;
   transition: border-color 80ms linear;
   white-space: nowrap;
```
Comment (MadeByMike) [inline packages/app-admin-ui/client/pages/Home/components.js]:
```suggestion
  white-space: nowrap;
  padding-right: 2em;
```

Comment (MadeByMike) [reply packages/app-admin-ui/client/pages/Home/components.js]: Allows text ellipsis to work

File: packages/app-admin-ui/client/pages/Home/components.js (hunk 2 above)
Comment (Vultraz) [reply packages/app-admin-ui/client/pages/Home/components.js]: `overflow-y` is what caused the letter bottoms to be cut off. I stuck with `overflow-x` so long labels wouldn't extend past the right edge of the card.

File: packages/app-admin-ui/client/pages/Home/components.js (hunk 3 above)
Comment (Vultraz) [reply packages/app-admin-ui/client/pages/Home/components.js]: Added. Needed to switch to a div for it to work, though (and I still needed `overflow-x`).

File: packages/app-admin-ui/client/pages/Home/components.js (hunk 2 above)
Comment (MadeByMike) [reply packages/app-admin-ui/client/pages/Home/components.js]: It was really the line-height that was causing the issue not the overflow.

Comment (Vultraz) [reply packages/app-admin-ui/client/pages/Home/components.js]: Ah, you're right! Done.

File: packages/app-admin-ui/client/pages/Home/components.js (hunk 3 above)
Comment (MadeByMike) [reply packages/app-admin-ui/client/pages/Home/components.js]: Add `box-sizing: border-box` rather than changing it to a div.

Comment (Vultraz) [reply packages/app-admin-ui/client/pages/Home/components.js]: Done.

**Comment by Vultraz** [comment]
Ok, I don't understand why the tests are failing now :|

**Pull request merged**